### PR TITLE
Implements UnassignRoleActivity

### PR DESCRIPTION
**Pull request by HermesSbicego-Laser**
UnassignRoleActivity #7396
- created the Activity
- defined the css and the view
- moved the private function GetRoles in order to reuse it in both AssignRoleActivity and UnassignRoleActivity

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            _roleService = roleService;
+            T = NullLocalizer.Instance;
+            Logger = NullLogger.Instance;
+        }
+
+        public Localizer T { get; set; }
+        public ILogger Logger { get; set; }
+
+        public override string Name
+        {
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs]: Formatting

File: src/Orchard.Web/Modules/Orchard.Roles/Activities/AssignRoleActivity.cs [hunk 2]
```diff
@@ -78,15 +78,5 @@ public class AssignRoleActivity : Task {
 
             yield return T("Done");
         }
-        
-        private IEnumerable<string> GetRoles(ActivityContext context) {
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/AssignRoleActivity.cs]: Just copy-paste

File: src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs (hunk 1 above)
Comment (HermesSbicego-Laser) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs]: I don't know why, I have Rebracer installed (It uses rebracer.xml of the orchard.sln) but seems to be configured to use new line for get/set and properties...
is there a reason for that? It's a rebracer settings regression?

Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs]: It works fine for me, maybe a setting, or another plugin taking over (Resharper?)

Comment (HermesSbicego-Laser) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs]: I do not have any other extension on my VS
this is what I have in Rebracer.xml under "src". In particular         **NewLines_Braces_Property 1** Is that correct?

```
        <PropertyValue name="NewLines_AnonymousTypeInitializer_EachMember">1</PropertyValue>
        <PropertyValue name="NewLines_Braces_Accessor">1</PropertyValue>
        <PropertyValue name="NewLines_Braces_AnonymousMethod">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_AnonymousTypeInitializer">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_ArrayInitializer">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_CollectionInitializer">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_ControlFlow">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_LambdaExpressionBody">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_Method">0</PropertyValue>
        <PropertyValue name="NewLines_Braces_ObjectInitializer">0</PropertyValue>

        <PropertyValue name="NewLines_Braces_Property">1</PropertyValue>

        <PropertyValue name="NewLines_Braces_Type">0</PropertyValue>
        <PropertyValue name="NewLines_Keywords_Catch">1</PropertyValue>
        <PropertyValue name="NewLines_Keywords_Else">1</PropertyValue>
        <PropertyValue name="NewLines_Keywords_Finally">1</PropertyValue>
        <PropertyValue name="NewLines_ObjectInitializer_EachMember">1</PropertyValue>
        <PropertyValue name="NewLines_QueryExpression_EachClause">1</PropertyValue>

```

File: src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        public override string Name {
+            get { return "UnassignRole"; }
+        }
+
+        public override LocalizedString Category {
+            get { return T("User"); }
+        }
+
+        public override LocalizedString Description {
+            get { return T("Unassign specific roles to the current content item if it's a user."); }
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs]: ` ... from the current ...`

Comment (HermesSbicego-Laser) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/UnassignRoleActivity.cs]: Oh, yes... I will push the fix next monday

File: src/Orchard.Web/Modules/Orchard.Roles/Activities/AssignRoleActivity.cs [hunk 4]
```diff
@@ -15,8 +15,7 @@ namespace Orchard.Roles.Activities {
     [OrchardFeature("Orchard.Roles.Workflows")]
     public class AssignRoleActivity : Task {
         private readonly IWorkContextAccessor _workContextAccessor;
-        private readonly IRepository<UserRolesPartRecord> _repository;
-        private readonly IRoleService _roleService;
+        private readonly IRepository<UserRolesPartRecord> _repository; private readonly IRoleService _roleService;
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Roles/Activities/AssignRoleActivity.cs]: Two lines

**Pull request merged**